.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       max-parallel: 4
       matrix:
         python-version: ['3.6', '3.7', '3.8', '3.9']
-        wagtail-version: ['2.9', '2.10', '2.11', '2.12']
+        wagtail-version: ['2.9', '2.10', '2.11', '2.12', '2.13']
```

Comment:
```suggestion
        wagtail-version: ['2.11', '2.12', '2.13', '2.14']
```

Reply:
This will require a bit more changes because there are compability issues with `wagtail_grapple[channels]` since django 3.0 is required for wagtail 2.14 (https://docs.wagtail.io/en/latest/releases/2.14.html) 

But I think it is worth the change, thanks for the suggestion